config/game_options.txt

```diff
@@ -201,13 +201,6 @@ MIDROUND_ANTAG WIZARD
 ## The amount of time it takes for the emergency shuttle to be called, from round start.
 SHUTTLE_REFUEL_DELAY 12000
 
-## Variables calculate how number of antagonists will scale to population.
-## Used as (Antagonists = Population / Coeff)
-## Set to 0 to disable scaling and use default numbers instead.
-TRAITOR_SCALING_COEFF 6
```

Comment:
Why remove these ?

Reply:
They are no longer in use. The rulesets now have fixed amounts of antags per pop incide instead, as well as individual scaling probability, antag cap and scaling cost configs in the dynamic.json file.

Reply:
These are not dynamic only though.

Reply:
Oh god oh fuck. Reverted, thanks!